packages/core/src/lib/TwaManifest.ts

```diff
@@ -183,6 +183,16 @@ export class TwaManifest {
       appVersion: this.appVersionName,
       webManifestUrl: this.webManifestUrl ? this.webManifestUrl.toString() : undefined,
     });
+  }
+
+  /**
+   * Saves the TWA Manifest to the file-system.
+   *
+   * @param {String} filename the location where the TWA Manifest will be saved.
+   */
+  async saveToFile(filename: string): Promise<void> {
+    console.log('Saving Config to: ' + filename);
```

Comment:
Did you mean to leave this in? If so, should we use some of our logging infrastructure instead?

Reply:
I didn't write this method. It was there and I just used it